adapters/sharethrough/butler.go

```diff
@@ -71,6 +72,7 @@ func (s StrOpenRTBTranslator) requestFromOpenRTB(imp openrtb.Imp, request *openr
 	headers := http.Header{}
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
+	headers.Add("Accept-Encoding", "gzip")
```

Comment:
Are you sure this is necessary? I believe the default Go transport will set this for you and transparently handle the un-zipping when reading the response.

Reply:
My testings showed that if I'm not adding the header here, the ad server does not receive it. I don't think Go alters the request.